admin_manual/maintenance/docker/index.rst

```diff
@@ -0,0 +1,101 @@
+======
+Docker
+======
+
+ownCloud can be installed using Docker, using `the official ownCloud Docker image <https://hub.docker.com/r/owncloud/server/>`_.
+This official image is designed to work with a data volume in the host filesystem and with separate *MariaDB* and *Redis* containers.
+The configuration:
+
+- exposes ports 80 and 443, allowing for HTTP and HTTPS connections.
+- mounts the data and MySQL data directories on the host for persistent storage.
+
+Installation on a Local Machine
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To use it, first create a new project directory and download ``docker-compose.yml`` from `the ownCloud Docker GitHub repository <https://github.com/owncloud-docker/server.git>`_ into that new directory.
+Next, create a `.env` configuration file, which contains the required configuration settings.
+Only a few settings are required, these are:
+
+================== ============================== =============
+Setting Name       Description                    Example
+================== ============================== =============
+``VERSION``        The ownCloud version           ``latest``
+``DOMAIN``         The ownCloud domain            ``localhost``
+``ADMIN_USERNAME`` The admin username             ``admin``
+``ADMIN_PASSWORD`` The admin user's password      ``admin``
+``HTTP_PORT``      The HTTP port to bind to       ``80``
+``HTTPS_PORT``     The HTTP port to bind to       ``443``
+================== ============================== =============
+
+Then, you can start the container, using your preferred Docker command-line tool. 
+The example below shows how to use `Docker Compose <https://docs.docker.com/compose/>`_. 
+
+.. note:: 
+   You can find instructions for using plain docker `in the GitHub repository <https://github.com/owncloud-docker/server#launch-with-plain-docker>`_.
+
+.. code-block:: console
+
+   # Create a new project directory
+   mkdir owncloud-docker-server
+
+   cd owncloud-docker-server
+
+   # Copy docker-compose.yml from the GitHub repository
+   wget https://github.com/owncloud-docker/server/blob/master/docker-compose.yml
```

Comment:
This must be https://raw.githubusercontent.com/owncloud-docker/server/master/docker-compose.yml